test/actions/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 project(Actions_Test)
 
 add_executable(Actions_test Actions_test.cpp)
-target_link_libraries(Actions_test atlascore atlasinsts SPARTA::sparta mavis)
+target_link_libraries(Actions_test atlascore atlasinsts ${SPARTA} mavis)
```

Comment:
Can all these be SPARTA_LIBS instead of SPARTA?

Reply:
Yep, I fixed it!